wordpress-mobile/WordPress-iOS
# Transitioned Latest Post chart to leverage actual data

Fixes #11067.

To test:
- Checkout the branch & confirm that existing tests pass.
- Navigate to Stats for a given site. Observe the Latest Post Summary now reflects live data (i.e., similar aesthetics to #11061, up to 14 bars, not tappable).
- For the same site as in the preceding step, view stats for the same selected post (1 of 3 ways, as in #11605). Note the bars are still tappable (as in #11605), and the shape of the graph is the same.

Update release notes:


| Latest Post Chart | Selected Post Chart |
| - | - |
| <img width="584" alt="11067a" src="https://user-images.githubusercontent.com/221062/57093675-632fa480-6cc3-11e9-88cc-16dc11fff4e6.png"> | <img width="584" alt="11067b" src="https://user-images.githubusercontent.com/221062/57093676-632fa480-6cc3-11e9-8c6b-e3eaa0c31b2f.png"> |
